### ISSUE TITLE
ADC API response spec/docs inconsistency

### ISSUE THREAD

**Issue by bcorrie** [airr-community/airr-standards#227]
Hi @schristley 
In looking at the ADC API spec, it looks like if a POST query on /repertoire or /rearrangement succeeds (returns 200) it should always return a response that looks like {"Info":{}, "Repertoire":[]} or  {"Info":{}, "Rearrangement":[]} respectively.

The example on the docs page here: http://docs.airr-community.org/en/metadata-docs/api/overview.html for example query query1-2_repertoire.json has a response that looks like this: { "success":true, "result": [] }

I think the docs are older and need to be corrected in this case, is that right?

**Comment by bcorrie** [comment]
Oh yes, VDJServer's ADC implementation responds correctly with {"Info":{}, "Repertoire":[]}

**Comment by bcorrie** [comment]
In fact, I think the fix for the docs should be:
```
{
    "Info": {
        "title": "AIRR Data Commons API",
        "description": "API response for repertoire query",
        "version": 1.3,
        "contact": {
            "name": "AIRR Community",
            "url": "https://github.com/airr-community"
        }
    },
    "Repertoire": [
        {
            "repertoire_id": "4357957907784536551-242ac11c-0001-012"
        },
        {
            "repertoire_id": "4476756703191896551-242ac11c-0001-012"
        },
        {
            "repertoire_id": "6205695788196696551-242ac11c-0001-012"
        },
        {
            "repertoire_id": "6393557657723736551-242ac11c-0001-012"
        },
        {
            "repertoire_id": "7158276584776536551-242ac11c-0001-012"
        }
    ]
}
```

**Comment by bcorrie** [comment]
Let me know if I got this right and I can update the docs...

**Comment by schristley** [comment]
Yeah the docs are old (already!) in a few places. Bojan pointed out #225 which means the data needs to be updated, re-loaded and the docs updated with the right response as well.

Please, yes, update the docs! If you could, also write a short segment on `Info` because that was added late. `Info` is same structure as for the OpenApi V2, and what we have in AIRR Schema. The key point is the `version` should be the AIRR Schema version that the returned data conforms too. That way tools which download the data can know that, though of course it's optional (we haven't really discussed in CRWG if it should be required).

**Comment by bcorrie** [comment]
@schristley fixed the above docs and the docs around #225. 

Added an info_object in the spec and made the rearrangement_response and repertoire_response refer to it...

Also addressed #228 so both /repertoire/{id} and /rearrangment/{id} return consistent repertoire_response and rearrangement_response.
